BoldGrid/weforms
# fixes #89

Fixes issue #149  with placeholder text being replaced with a blank value.

also fixes https://github.com/BoldGrid/weforms-pro/issues/89.

replace tag runs on both notifications and front-end rendering. When rendering on the front end there is no entry id, unlike with notifications from an entry.

To test:
Create a form with placeholder text on name field or textarea field
ensure placeholder shows on frontend.
